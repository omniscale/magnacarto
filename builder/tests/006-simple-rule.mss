#class {
    line-width: 12;
    line-color: #ff0
}

#class-bar {
    line-width: 12;
    line-color: #ff0
}
