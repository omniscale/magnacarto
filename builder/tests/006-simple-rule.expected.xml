<Map srs="epsg:3857">
  <Parameters></Parameters>
  <Style name="class" filter-mode="first">
    <Rule>
      <LineSymbolizer stroke="#ffff00" stroke-width="12"></LineSymbolizer>
    </Rule>
  </Style>
  <Style name="class-bar" filter-mode="first">
    <Rule>
      <LineSymbolizer stroke="#ffff00" stroke-width="12"></LineSymbolizer>
    </Rule>
  </Style>
  <Layer name="class" srs="" status="off">
    <StyleName>class</StyleName>
  </Layer>
  <Layer name="class-bar" srs="" status="off">
    <StyleName>class-bar</StyleName>
  </Layer>
</Map>